### PR TITLE
feat: frontend support for login rule audit events

### PR DESCRIPTION
**Pull request by nklaassen**
This PR adds frontend support to nicely display login rule audit events. I followed the guide [here](https://github.com/gravitational/teleport/tree/master/web#adding-an-audit-event) and mostly copied the types used for the role create/delete events.

~So far I've skipped steps 5 and 6 in the guide which add example events to the storybook because the storybook is currently broken since the webapps merge, I can add those once it is fixed.~ Example events have now been added to the test fixtures. `yarn test && yarn lint` passes locally.

Screenshot 2023-01-25 at 11 22 25 AM

### CODE REVIEW

**Comment by nklaassen** [comment]
> Adding the example events to https://github.com/gravitational/webapps/blob/master/packages/teleport/src/Audit/fixtures/index.ts also includes them in the tests, so it might be worth adding them for now

The tests fail without `yarn test-update-snapshot` which currently doesn't exist in the project root and I'm not sure I can run it in the web/ subdir? Sounds like @hatched is posting a storybook fix pretty soon so I'll hold off merging this until I can add that.

**Review by zmb3 — APPROVED** [review]
LGTM pending the storybook part

**Comment by ryanclark** [comment]
> The tests fail without `yarn test-update-snapshot` which currently doesn't exist in the project root and I'm not sure I can run it in the web/ subdir? Sounds like @hatched is posting a storybook fix pretty soon so I'll hold off merging this until I can add that.

`yarn test -u` from the root works

File: web/packages/teleport/src/Audit/fixtures/index.ts [hunk 1]
```diff
@@ -2671,6 +2671,28 @@ export const events = [
...
+    event: 'login_rule.create',
+    expires: '0001-01-01T00:00:00Z',
+    name: 'test_rule',
+    time: '2023-01-25T19:21:36.144Z',
+    uid: '266e8563-729e-412f-ba26-1050fbec0cd6',
+    user: 'nic',
+  },
+  {
+    cluster_name: 'im-a-cluster-name',
+    code: 'TLR00I',
```
Comment (ryanclark) [inline web/packages/teleport/src/Audit/fixtures/index.ts:2687]:
```suggestion
    code: 'TLR01I',
```

Comment (nklaassen) [reply web/packages/teleport/src/Audit/fixtures/index.ts]: 🤦‍♂️ thanks, fixed

**Review by ryanclark — APPROVED** [review]

**Pull request merged**